Source/Atomic/Atomic2D/ParticleEmitter2D.cpp

```diff
@@ -323,7 +326,7 @@ void ParticleEmitter2D::Update(float timeStep)
 
 bool ParticleEmitter2D::EmitParticle(const Vector3& worldPosition, float worldAngle, float worldScale)
 {
-    if (numParticles_ >= (unsigned)effect_->GetMaxParticles() || numParticles_ >= particles_.Size())
+    if (!emitting_ || numParticles_ >= (unsigned)effect_->GetMaxParticles() || numParticles_ >= particles_.Size())

```

Comment:
Yea, missed that... next time I'm gonna run a comparer to make sure I'm not forgetting guards.